### PR TITLE
feat: make parseWhole available only for "options"

### LINKED ISSUE #121: Make field parseWhole available only for options section
Now, field parseWhole is available for all sections. Because sections usage and examples are not required and many cli doesn't have it, field parseWhole should be available only for options section [required section]. 

Please, make parseWhole field available only for options section.

### PR DESCRIPTION
**Pull request by ifedchankau**
Proposed Changes:

- Make field parseWhole in config available only for section "options"

Closes #121

### CODE REVIEW

File: src/template/configSchema.json [hunk 1]
```diff
@@ -82,6 +82,19 @@
...
+                }
+            },
+            "additionalProperties": false
+        },
+        "section:required": {
+            "title": "Required section",
+            "description": "Required section of help",
+            "properties": {
+                "names": {
+                    "$ref": "#/definitions/section:optional/properties/names"
```
Comment (romanrostislavovich) [inline src/template/configSchema.json]: I suggest to move following properties to `root` lvl of definitions: `names` and `postfix`

File: src/template/configSchema.json [hunk 2]
```diff
@@ -82,6 +82,19 @@
                             "$ref": "#/definitions/default:array"
                         }
                     ]
+                }
+            },
+            "additionalProperties": false
+        },
+        "section:required": {
+            "title": "Required section",
+            "description": "Required section of help",
```
Comment (romanrostislavovich) [inline src/template/configSchema.json]: maybe you meant the section of `configuration`?

File: src/template/configSchema.json (hunk 1 above)
Comment (ifedchankau) [reply src/template/configSchema.json]: @svoboda-rabstvo done.

File: src/template/configSchema.json (hunk 2 above)
Comment (ifedchankau) [reply src/template/configSchema.json]: @svoboda-rabstvo yes, done.

**Review by ghost — APPROVED** [review]

**Pull request merged**